logging/google/cloud/logging/_shutdown.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for code related to shutdown logging."""
+
+import functools
+import os
+import signal
+import sys
+import traceback
+
+from google.cloud.logging._environment_vars import APPENGINE_FLEXIBLE_ENV_VM
+from google.cloud.logging._environment_vars import APPENGINE_FLEXIBLE_ENV_FLEX
+from google.cloud.logging._environment_vars import GAE_SERVICE
+from google.cloud.logging._environment_vars import GAE_VERSION
+from google.cloud import _helpers
+
+import six
```

Comment:
Just noticed the location of this. It's 3rd party, so it should go above the `google.cloud` imports